lib/twitter_list.rb

```diff
@@ -1,12 +1,8 @@
-# require 'bundler/setup'
-# require 'scraperwiki'
-# require 'twitter'
-# require 'pry'
+require 'bundler/setup'
+require 'twitter'
+require 'dotenv'
```

Comment:
This isn't going to work when other people try to install the gem because `dotenv` is currently only a development dependency in the gemspec. It needs to be a runtime dependency:

```diff
diff --git a/twitter_list.gemspec b/twitter_list.gemspec
index 4c2b3d0..d5981b2 100644
--- a/twitter_list.gemspec
+++ b/twitter_list.gemspec
@@ -20,11 +20,12 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
   s.date          = '2016-07-18'
 
+  s.add_runtime_dependency 'dotenv'
+
   s.add_development_dependency 'bundler', '~> 1.11'
   s.add_development_dependency 'rake', '~> 11.2'
   s.add_development_dependency 'rspec', '>=3.4.0'
   s.add_development_dependency 'coveralls'
   s.add_development_dependency 'pry', '~> 0.10'
-  s.add_development_dependency 'dotenv'
   s.add_development_dependency 'twitter'
 end
```

However, I'm not sure having `dotenv` as a runtime dependency really makes sense either - people won't necessarily want to use a `.env` file for their environment variables. I think it probably makes sense to just drop the `dotenv` dependency for now and use something like [autoenv](https://github.com/kennethreitz/autoenv) to set env vars in your shell for testing purposes.